extensions/wikia/CreatePage/CreatePage.php

```diff
@@ -162,7 +162,7 @@ function wfCreatePageAjaxGetVEDialog() {
 
 	$body['html'] = $template->render( 'dialog-ve' );
 	$body['title'] = wfMsg( 'createpage-dialog-title' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)